webpack/components/extensions/RegistrationCommands/fields/ActivationKeys.js

```diff
@@ -41,7 +41,12 @@ const ActivationKeys = ({
   useEffect(() => {
     handleInvalidField('Activation Keys', akHasValidValue(hostGroupId, pluginValues?.activationKeys, hostGroupActivationKeys));
   }, [handleInvalidField, hostGroupId, hostGroupActivationKeys, pluginValues]);
-
+  
+  useEffect(() => {
+    if(activationKeys.length==1){
```

Comment:
```suggestion
    if (activationKeys.length === 1) {
```

Reply:
@girijaasoni `npm run lint -- --fix` is your friend

Reply:
Thanks @stejskalleos